backend/src/bin/web/main.rs

```diff
@@ -283,6 +283,10 @@ async fn client_msg(
                     let res = listen_for_web(req.clone()).await;
                     reply(req, client, res)
                 }
+                Commands::NFCGet | Commands::NFCSet => {
```

Comment:
if you aren't doing anything new you can join it with the door look commands above using |